internal/hostport/hostport.go

```diff
@@ -52,7 +52,17 @@ type PodPortMapping struct {
 	IP           net.IP
 }
 
+// ipFamily refers to a specific family if not empty, i.e. "4" or "6".
+type ipFamily string
+
+// Constants for valid IPFamily:
+const (
```

Comment:
So, it seems that this commit is too overloaded, i.e. it does too many things:
 - adds IPv6 support
 - fixes nits found by golangci-lint
 - adds tests for hostport pkg
 - switches cri-o to use the new hostport implementation

Perhaps it'll be better to split it into separate commits, in the following order:
1. fix golangci nits
2. add IPv6 support
3. add unit tests
4. switch crio to new hostport

or something like that

Reply:
can we go with current status this time? :sweat_smile:

Reply:
:+1:

Reply:
@kolyshkin I split it out, with a small different approach, please let me know if it works for you